utils/util_material.py

```diff
@@ -17,16 +17,16 @@ class FaceMap(AutoIndex):
     SLABS = auto()
     WALLS = auto()
 
+    FRAME = auto()
+
```

Comment:
Having a single frame option for everything will give uses restrictions when adding materials. It may be useful to have different facemaps for windows and doors.

Reply:
Assigning different facemaps to frame faces in a single component will be difficult and confusing.

Also, I can't think of any use-case when someone would need different material for frames of window and door.

Reply:
> Assigning different facemaps to frame faces in a single component will be difficult and confusing.

Yes it is. Keeping the geometry generation routines in different functions simplifies things.

> Also, I can't think of any use-case when someone would need different material for frames of window and door.

My hope was not to limit the choices users have.
We can keep it simple for now and hope that users will not want different facemaps.